### ISSUE TITLE
Make the webpage better

### ISSUE THREAD

**Issue by jluims** [jluims/RobloxGen#5]
# Right now it's sorta ugly
TODO:
- [ ] Make it look nicer
- [ ] Add statistics (accounts created, accounts per day, etc.)

**Comment by Arcxdd** [comment]
You want me to write you a whole RestAPI and Mobile app with ReactNative too?

**Comment by jluims** [comment]
RestApi sounds a little overkill for the scope of the project imo. But the thing could use a mobile app

**Comment by Arcxdd** [comment]
restapi's are easy to do though

**Comment by Arcxdd** [comment]
and pretty easy to hold up

**Comment by jluims** [comment]
using express or another lib?

**Comment by Arcxdd** [comment]
any library really, ive kind of used them all

**Comment by jluims** [comment]
Have any example repos for REST api in node.js? (doesn't have to be made by you) @Arcxdd

**Comment by Arcxdd** [comment]
I do have repositories of REST APIs, but they're all private due to confidential information being stored in them. (ex. tokens, passwords, databases)

TLDR; no 😭

**Comment by Arcxdd** [comment]
And I'm too lazy to find an example.

**Comment by jluims** [comment]
go for it if it makes things any better 😭

**Comment by Arcxdd** [comment]
ok let me just fork & rewrite your whole project and turn it into a scalable rest api :)

**Comment by jluims** [comment]
i'm not that familiar with git, can't you use a new branch for that?

**Comment by Arcxdd** [comment]
well yes but id rather fork

**Comment by Arcxdd** [comment]
im not actually gonna do that



at least not yet

**Comment by jluims** [comment]
I don't mind, the current http api is doing okay

**Comment by WaltDisneyWorld** [comment]
Truthfully, I could also contribute.

**Comment by WaltDisneyWorld** [comment]
Do you have discord?

**Comment by jluims** [comment]
@WaltDisneyWorld https://github.com/jluims/jluims/blob/main/README.md

**Comment by WaltDisneyWorld** [comment]
```import dotenv from 'dotenv'
       ^^^^^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:721:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)```
